src/Build/Resources/Strings.resx

```diff
@@ -1707,13 +1707,16 @@ Utilization:          {0} Average Utilization: {1:###.0}</value>
       LOCALIZATION:  Do not localize the following words: ProjectGraph, ProjectReference, ToolsVersion.
     </comment>
   </data>
+  <data name="CircularDependencyInProjectGraph" UESanitized="false" Visibility="Public">
+    <value>MSB4251: Exception occurred while creating depenendency graph: There is a circular dependency involving the following projects: {0}</value>
+  </data>
```

Comment:
I think you need the `<comment>` element to explain to the Localizers what to do. Follow the pattern used in some of the other errors. The "CircularDependency" string in particular seems like a good one to base off of in this case.